services/src/main/scala/org/apache/gearpump/services/RestServices.scala

```diff
@@ -20,36 +20,37 @@ package org.apache.gearpump.services
 
 import akka.actor.{Actor, ActorRef, ActorSystem, Props}
 import akka.io.IO
-import org.apache.gearpump.services.WebSocketServices._
+import com.typesafe.config.Config
 import org.apache.gearpump.util.LogUtil
 import spray.can._
 import spray.routing.RoutingSettings
 
+import scala.concurrent.ExecutionContextExecutor
+
 trait RestServices extends AppMastersService
     with AppMasterService with WorkerService with WorkersService with MasterService
     with ConfigQueryService with MetricsQueryService with WebSocketService with StaticService {
-  implicit def executionContext = actorRefFactory.dispatcher
+  implicit def executionContext: ExecutionContextExecutor = actorRefFactory.dispatcher
 
   lazy val routes = appMastersRoute ~ appMasterRoute ~ workersRoute ~ workerRoute ~
     masterRoute ~  configQueryRoute ~ metricQueryRoute ~ webSocketRoute ~ staticRoute
 }
 
-class RestServicesActor(masters: ActorRef, sys:ActorSystem) extends Actor with RestServices {
+class RestServicesActor(masters: ActorRef, sys:ActorSystem, conf:Config) extends Actor with RestServices {
   def actorRefFactory = context
```

Comment:
conf can be retrieved by system.settings.config

Do we really need to add an extra parameter?